### PR TITLE
data:rajmaurya-dev

### PR DESCRIPTION
**Pull request by rajmaurya-dev**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4292"></a>

### CODE REVIEW

**Review by arpitghura — DISMISSED** [review]
Looks Good

File: data/rajmaurya-dev.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "bio": "Frontend Developer | Open source contributor | Building Projects",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rajmaurya_dev",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/rajmaurya-dev/",
```
Comment (Panquesito7) [inline data/rajmaurya-dev.json:14]: Not sure the `www.` is needed here.
```suggestion
      "url": "https://github.com/rajmaurya-dev/",
```

**Review by stevenovak123 — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**